backend/src/routes/user.js

```diff
@@ -1,18 +1,60 @@
 const express = require("express");
+var multer = require("multer");
+var fs = require("fs");
+var path = require("path");
 
 const router = express.Router();
+const upload = multer();
 
-router.get("/:userId", (req, res) => {
-  res.send({
-    firstName: "Johnny",
-    lastName: "tester",
-    email: "123@fake.com",
-    aspirationalMessage: "If you can dream it, you can do it!",
-  });
+router.get("/:userId", async (req, res) => {
+  const user = await req.context.models.User.findById(req.params.userId);
+  const userData = user.dataValues;
+
+  // Delete extraneous data
+  delete userData.createdAt;
+  delete userData.updatedAt;
+
+  res.send(userData);
+});
+
+router.get("/avatar/:userId", (req, res) => {
+  let filePath = path.join(__dirname, `../../data/${req.params.userId}.jpg`);
+  if (fs.existsSync(filePath)) res.sendFile(filePath);
+  else res.sendStatus(404);
 });
 
-router.post("/", (req, res) => {
-  res.send("Placeholder");
+router.post("/", upload.single("avatar"), async (req, res) => {
+  // Return error if data is missing
+  let fileName = false;
+  if (
+    !req.body.firstName ||
+    !req.body.lastName ||
+    !req.body.email ||
+    !req.body.aspirationalMessage ||
+    !req.file
```

Comment:
Not sure avatars should be mandatory to make an account. But maybe we can just send a default icon if the user does not opt to add any.

Reply:
I'll add a temp one, like the egg on twitter.